utils/restic/Makefile

```diff
@@ -1,12 +1,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=restic
-PKG_VERSION:=0.9.6
-PKG_RELEASE:=2
+PKG_VERSION:=0.13.1
+PKG_RELEASE:=3
```

Comment:
Please reset `PKG_RELEASE` to `1` or set to `$(AUTORELEASE)`.

Reply:
will do, I am still struggling with git